akka-docs/src/test/java/jdocs/stream/operators/SourceDocExamples.java

```diff
@@ -32,8 +37,10 @@
   public static final TestKitJunitResource testKit = new TestKitJunitResource(ManualTime.config());
 
   public static void fromExample() {
+    final ActorSystem theSystem = null;
+
     // #source-from-example
-    final ActorSystem system = ActorSystem.create("SourceFromExample");
+    final ActorSystem system = theSystem;
```

Comment:
I don't think this is an improvement, perhaps just move it out of the sample snippet instead?

Reply:
what I wanted to avoid is that it is copied and thereby creating a new ActorSystem for each stream 😨 

maybe we explain enough in other places that an ActorSystem is needed?

Reply:
I think so